cvereporter/nist_enhance.py

```diff
@@ -0,0 +1,48 @@
+from cyclonedx.model.impact_analysis import ImpactAnalysisAffectedStatus
+from cyclonedx.model.vulnerability import Vulnerability, VulnerabilitySource,VulnerabilityScoreSource, VulnerabilityRating, VulnerabilitySeverity, BomTarget, BomTargetVersionRange
+import requests
+import json
+
+def fetch_nist(url: str) -> dict:
+    data = None
+    nist_resp = requests.get(url)
+    if nist_resp.status_code != 200:
+        print("error fetching "+id)
+    else:
+        data = nist_resp.json()
+    return data
+def extract_relevant_parts(nist_resp: dict) -> dict:
+    # todo: this can use a unit test at some point
+    ratings = []
+
+    cve = nist_resp["vulnerabilities"][0]["cve"]
+
+    for metrics in cve["metrics"]["cvssMetricV31"]:
+        #todo: do we need recommendations from NIST as well?
+        relevant = {}
+        relevant["source"] = metrics["source"]
```

Comment:
hi @tellison is this the kind of intermediate data structure you were thinking about for representing the data before populating it into the BOM itself? (I know this is on the nist side, i can eventually move the ojvg side to a similar thing as well)